nixos/modules/services/networking/speedify.nix

```diff
@@ -0,0 +1,59 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.speedify;
+in
+{
+  options.services.speedify = {
+    enable = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      description = ''
+        This option enables Speedify daemon.
+        This sets {option}`networking.firewall.checkReversePath` to "loose", which might be undesirable for security.
+      '';
+    };
+
+    package = lib.mkPackageOption pkgs "speedify" { };
+  };
+
+  config = lib.mkIf cfg.enable {
+    boot.kernelModules = [ "tun" ];
+
+    networking.firewall.checkReversePath = "loose";
+
+    systemd.services.speedify = {
+      description = "Speedify Service";
+      wantedBy = [ "multi-user.target" ];
+      wants = [
+        "network.target"
+        "network-online.target"
+      ];
+      after = [
+        "network-online.target"
+        "NetworkManager.service"
```

Comment:
Should this be made conditional on networkmanager being enabled?

Reply:
Mullvad has it non-conditional https://github.com/NixOS/nixpkgs/blob/2aa78e24b5bab00270924557648e65380c291644/nixos/modules/services/networking/mullvad-vpn.nix#L58-L62

I guess we can change it to
```
after = [
  "network-online.target"
] ++ lib.optional config.networking.networkmanager.enable "NetworkManager.service"
  ```

Reply:
Seems to work as expected, I added it to the PR